### PR TITLE
Fix similarity correlation

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>
/kind improvement
/cc @cydrain

### CODE REVIEW

File: internal/core/src/segcore/SimilarityCorelation.h [hunk 1]
```diff
@@ -17,8 +17,7 @@ namespace milvus {
 namespace segcore {
 static inline bool
 PositivelyRelated(faiss::MetricType metric_type) {
-    return metric_type == faiss::MetricType::METRIC_INNER_PRODUCT || metric_type == faiss::MetricType::METRIC_Jaccard ||
-           metric_type == faiss::MetricType::METRIC_Tanimoto;
+    return metric_type == faiss::MetricType::METRIC_INNER_PRODUCT;
 }
 }  // namespace segcore
```
Comment (cydrain) [inline internal/core/src/segcore/SimilarityCorelation.h:22]: use
namespace milvus::segcore {

}

File: internal/proxy/task.go [hunk 2]
```diff
@@ -30,6 +30,8 @@ import (
 	"strings"
 	"unsafe"
 
+	"github.com/milvus-io/milvus/internal/util/distance"
+
```
Comment (cydrain) [inline internal/proxy/task.go:34]: please order these header files

Comment (DragonDriver) [reply internal/proxy/task.go]: Done.

File: internal/core/src/segcore/SimilarityCorelation.h (hunk 1 above)
Comment (DragonDriver) [reply internal/core/src/segcore/SimilarityCorelation.h]: Done.

**Comment by DragonDriver** [comment]
rerun failure checks

**Pull request merged**